javabuilder-authorizer/token_status.rb

```diff
@@ -0,0 +1,11 @@
+module TokenStatus
+  # Token statuses used in HTTP authorizer
+  VALID_HTTP = 'valid_http'.freeze
+  ALREADY_EXISTS = 'already_exists'.freeze
+
+  # Token statuses used in websocket authorizer
+  VALID_WEBSOCKET = 'valid_websocket'.freeze
+  UNKNOWN_ID = 'unknown_id'.freeze
+  NOT_VETTED = 'not_vetted'.freeze
+  USED = 'used'.freeze
```

Comment:
NIT: Mind adding a quick comment to each of these statuses explaining what they mean? It's clear now, but I'm guessing a little more explanation will help us out in the future.

Reply:
Can do!